components/chainhook-cli/src/config/generator.rs

```diff
@@ -19,12 +19,15 @@ mode = "{network}"
 bitcoind_rpc_url = "http://localhost:8332"
 bitcoind_rpc_username = "devnet"
 bitcoind_rpc_password = "devnet"
-# Bitcoin block events can be received by Chainhook
-# either through a Bitcoin node's ZeroMQ interface,
-# or through the Stacks node. The Stacks node is
-# used by default:
+
+# Chainhook must be able to receive Bitcoin block events.
+# These events can originate from either a Stacks node or a Bitcoin node's ZeroMQ interface.
+
+# By default, the service is set to receive Bitcoin block events from the Stacks node (via burnchain):
```

Comment:
"via burnchain" might beg some context... Burnchain is a deprecated PoX v1 term that persists in old Stacks Node software. Not even some (a lot of?) Stacks users might know what it is. Is there a better way to put it?